### ISSUE TITLE
3.2.0 crashes when scrolling the main window via mouse wheel

### ISSUE THREAD

**Issue by alexmurray** [alexmurray/shortwave-snap#7]
Running with `G_MESSAGES_DEBUG=all` we see the following output:

```
(shortwave:154475): Gdk-DEBUG: 14:25:27.345: interface 'wl_pointer' has no event 9

Gdk-Message: 14:25:27.345: Error reading events from display: Timer expired
```

**Comment by alexmurray** [comment]
This issue for the steam flatpak is very similar - https://steamcommunity.com/app/353380/discussions/10/3761104049339603835/ - but with no resolution.

I wonder if there is some library version mismatch...?

**Comment by alexmurray** [comment]
CC @driesdeschout

**Comment by alexmurray** [comment]
I am running Ubuntu 23.10 on an Intel Tiger Lake machine with integrated graphics and I still see this crash unfortunately, even after removing all data from `~/snap/shortwave`.  The patch to remove additional libraries sounds promising - thanks for these efforts.

**Comment by driesdeschout** [comment]
Could you test again with PR #8 applied?

NB: my testing was all done on Ubuntu 22.04.

**Comment by alexmurray** [comment]
Fixed via #8
